source/more-info/free-space.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: "Clear up storage"
+description: "More information on how to clear up storage in Home Assistant."
+---
+
+Reaching your storage limit sucks, this page will help you when that happens.
+
+There are several things you can do to free up some space:
+
+- [Clean the database](#clean-the-database)
+- [Delete old snapshots](#delete-old-snapshots)
+- [Uninstall unused add-ons](#uninstall-unused-add-ons)
+- [Last resort](#last-resort)
+
+## Clean the database
+
+The Home Assistant database can get huge!
+
+Luckily there is a tool you can use to [purge the contents of the database](https://www.home-assistant.io/integrations/recorder/#service-purge)
+
+You can also [filter](https://www.home-assistant.io/integrations/recorder/#configure-filter)
+what you send to the database.
+
+## Delete old snapshots
+
+Open the Home Assistant UI and go to "Supervisor" in the sidebar, and then to
+the "Snapshots" tab, here you will see all your snapshots, these can be
+downloaded and placed somewhere safe, when you have done that, you can delete
+them in the UI and it will free up some space for you.
+
+## Uninstall unused add-ons
+
+Add-ons can take a lot of space, not just the add-on itself but also their data.
+
+Open the Home Assistant UI and go to "Supervisor" in the sidebar, you will be
+on the "Dashboard" tab when you click on the "Supervisor", here you will see
+all your installed add-ons, maybe you have some that you no longer use, if
+you do those can be uninstalled to free up some space.
+
+## Last resort
+
+If all else fails, you need to expand your storage.
+
+If you are running Home Assistant as a VM, you can just give it more and on the
```

Comment:
At least remove the "just" part. As it might not be that easy for everybody.

```suggestion
If you are running Home Assistant as a VM, you can give it more and on the
```

Reply:
Reworded

> If you are running Home Assistant as a VM, you can give it more, look at the
documentation for your hypervisor on how to expand disks for virtual machines
and on the next boot Home Assistant will auto-expand to use the newly added space.